### ISSUE TITLE
Create rule S6851 (`jsx-a11y/img-redundant-alt`): Images should have a non-redundant alternate description

### ISSUE THREAD

**Issue by ilia-kebets-sonarsource** [SonarSource/SonarJS#4380]
[plugin rule](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/img-redundant-alt.md#jsx-a11yimg-redundant-alt)
